### PR TITLE
CLOUD-830 Added quick install and uninstall steps to the doc

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   docs/monitor-kubernetes.md

### CODE REVIEW

**Review by fiowro — APPROVED** [review]
lgtm :)

File: docs/monitor-kubernetes.md [hunk 1]
```diff
@@ -22,52 +14,94 @@ To set up monitoring of Kubernetes, you need the following:
...
+
+      We recommend to use a separate namespace like `monitoring-system` for the same.    
+
+      ```{.bash data-prompt="$" }
+      $ curl -fsL  https://raw.githubusercontent.com/Percona-Lab/k8s-monitoring/main/vm-operator-k8s-stack/quick-install.sh | bash -s -- --api-key <API-KEY> --pmm-server-url <PMM-SERVER-URL> --k8s-cluster-id <UNIQUE-K8s-CLUSTER-IDENTIFIER> --namespace <NAMESPACE> 
+      ```    
+
+    !!! note    
+
+        The Prometheus node exporter is not installed by default since it requires privileged containers with the access to the host file system. If you need the metrics for Nodes, enable the Prometheus node exporter by setting the `prometheus-node-exporter.enabled` flag in the Victoria Metrics Helm chart to `true`. Refer to the [Install manually](install-the-victoria-metrics-kubernetes-monitoring-stack-manually) section.
```
Comment (cshiv) [inline docs/monitor-kubernetes.md:58]: For the quick install, user can just use another flag `--node-exporter-enabled` 
which in turn translates to 

```
curl -fsL  https://raw.githubusercontent.com/Percona-Lab/k8s-monitoring/main/vm-operator-k8s-stack/quick-install.sh | bash -s -- --api-key <API-KEY> --pmm-server-url <PMM-SERVER-URL> --k8s-cluster-id <UNIQUE-K8s-CLUSTER-IDENTIFIER> --namespace <NAMESPACE> --node-exporter-enabled
```

File: docs/monitor-kubernetes.md [hunk 2]
```diff
@@ -22,52 +14,94 @@ To set up monitoring of Kubernetes, you need the following:
...
-        $ API_KEY=$(curl --insecure -X POST -H "Content-Type: application/json" -d '{"name":"operator", "role": "Admin"}' "https://<login>:<password>@<server_host>/graph/api/auth/keys" | jq .key)
+        $ API_KEY=$(curl --insecure -X POST -H "Content-Type: application/json" -d {"name":"operator", "role": "Admin"}' "https://<login>:<password>@<server_host>/graph/api/auth/keys" | jq .key)
+            ```
+
+        !!! note
+
+            The API key is not rotated. 
+
+## Install the Victoria Metrics Kubernetes monitoring stack
+
```
Comment (cshiv) [inline docs/monitor-kubernetes.md:39]: ### Quick Install 
Maybe a header can be added

File: docs/monitor-kubernetes.md [hunk 3]
```diff
@@ -154,7 +188,7 @@ As a result, you have the `customresource-config-ksm` ConfigMap created.
          -n monitoring-system
```
Comment (cshiv) [inline docs/monitor-kubernetes.md:188]: ``` --set vmagent.spec.externalLabels.k8s_cluster_id=test-cluster> ```
Extra character > is present `test-cluster>`

File: docs/monitor-kubernetes.md [hunk 4]
```diff
@@ -154,7 +188,7 @@ As a result, you have the `customresource-config-ksm` ConfigMap created.
...
         ```
 
-    === "Configuration file" 
+    === ":octicons-file-code-24: Configuration file" 
 
          1. Edit the [`values.yaml`](https://raw.githubusercontent.com/Percona-Lab/k8s-monitoring/main/vm-operator-k8s-stack/values.yaml)
```
Comment (cshiv) [inline docs/monitor-kubernetes.md:193]: This section is required only if the user has not changed the values by using --set when installing helm chart.
For example we could achieve the same by using 
```
         --set externalVM.write.url=<PMM-SERVER-URL>/victoriametrics/api/v1/write \
         --set vmagent.spec.externalLabels.k8s_cluster_id=<UNIQUE-CLUSTER-IDENTIFER/NAME>
````
in the helm install command

**Review by cshiv — COMMENTED** [review]
Minor changes are put in the comments. It would be great if they are addressed.

File: docs/monitor-kubernetes.md [hunk 5]
```diff
@@ -229,3 +263,36 @@ As a result, you have the `customresource-config-ksm` ConfigMap created.
...
+## Uninstall Victoria metrics Kubernetes stack
+
+To remove Victoria metrics Kubernetes stack used for Kubernetes cluster monitoring, use the cleanup script. By default, the script removes all the [Custom Resource Definitions(CRD)](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/) and Secrets associated with the Victoria metrics Kubernetes stack. To keep the CRDs, run the script with the `--keep-crd` flag.
+
+=== ":material-file-remove-outline: Remove CRDs"
+
+    Replace the `<NAMESPACE>` placeholder with the namespace you specified during the Victoria metrics Kubernetes stack installation: 
+
+    ```{.bash data-prompt="$" }
+    $ curl -fsL  https://raw.githubusercontent.com/Percona-Lab/k8s-monitoring/main/vm-operator-k8s-stack/cleanup.sh | bash -s -- --namespace <NAMESPACE>
```
Comment (cshiv) [inline docs/monitor-kubernetes.md:275]: Made few changes in the uninstall script, command to run will also change now
```
bash <(curl -fsL https://raw.githubusercontent.com/Percona-Lab/k8s-monitoring/main/vm-operator-k8s-stack/cleanup.sh) --namespace <NAMESPACE>
```

File: docs/monitor-kubernetes.md [hunk 6]
```diff
@@ -229,3 +263,36 @@ As a result, you have the `customresource-config-ksm` ConfigMap created.
...
+    ```
+
+=== ":material-file-outline: Keep CRDs"
+
+    Replace the `<NAMESPACE>` placeholder with the namespace you specified during the Victoria metrics Kubernetes stack installation: 
+
+    ```{.bash data-prompt="$" }
+    $ curl -fsL  https://raw.githubusercontent.com/Percona-Lab/k8s-monitoring/main/vm-operator-k8s-stack/cleanup.sh | bash -s -- --keep-crd --namespace <NAMESPACE>
```
Comment (cshiv) [inline docs/monitor-kubernetes.md:283]: With the script changes, this command will also change
```
bash <(curl -fsL https://raw.githubusercontent.com/Percona-Lab/k8s-monitoring/main/vm-operator-k8s-stack/cleanup.sh) --namespace <NAMESPACE> --keep-crd
```

**Review by cshiv — COMMENTED** [review]
Changing instructions

File: docs/monitor-kubernetes.md (hunk 5 above)
Comment (nastena1606) [reply docs/monitor-kubernetes.md]: Updated

File: docs/monitor-kubernetes.md (hunk 4 above)
Comment (nastena1606) [reply docs/monitor-kubernetes.md]: UPdated. Removed config file confoguration section leaving only command line one

File: docs/monitor-kubernetes.md (hunk 3 above)
Comment (nastena1606) [reply docs/monitor-kubernetes.md]: Fixed

File: docs/monitor-kubernetes.md (hunk 2 above)
Comment (nastena1606) [reply docs/monitor-kubernetes.md:39]: Reworked to tabs

**Review by cshiv — APPROVED** [review]
LGTM

**Pull request merged**